homeassistant/components/climate/mill.py

```diff
@@ -100,6 +100,15 @@ def available(self):
         """Return True if entity is available."""
         return self._heater.available
 
+    @property
+    def state(self):
```

Comment:
The `state` property should not be overridden. The base climate class will handle that.

Reply:
The default state function, will not distinguish if the heater is actually heating or not. It will always say `heating` when the operation mode is heating.

This will add the same behavior as the generic thermostat: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/climate/generic_thermostat.py#L195

Reply:
You could use `self._heater.is_heating` in `device_state_attributes` for now.

Reply:
Yes, I do that.
But another user contacted me, and made me aware that the behaviour for Mill is different than for the Generic thermostat in the frontend. 
This change makes it the same behaviour as the Generic thermostat, and I think it maks more sense.

Reply:
That shouldn't have been added at that time and we shouldn't add more things that are against our current architecture until the new architecture is in place.

Reply:
Ok